src/features/contactmoment/ContactmomentVragenMenu.vue

```diff
@@ -60,7 +60,9 @@ async function waitForConfirmation() {
 <style lang="scss" scoped>
 .vragen-menu {
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(3rem, 1fr));
+  grid-template-columns: repeat(auto-fill, minmax(1.5rem, 1fr));
+  gap: var(--spacing-default);
```

Comment:
I think a simpler (easier to read and maintain) way to do this might be:

```
display: flex;
flex-wrap: wrap;
```

Reply:
I chose this specific layout because it makes the component a bit more independent from it's parent: it allows the columns to grow to make use of all the leftover space, which makes it look a bit more neat in certain conditions. However, this becomes less relevant now that the sidebar is a fixed size. I will have a look.

Reply:
By tweaking the sidebar width, I was able to make it fit. Thanks!